### ISSUE TITLE
Enhancement request: data/time

### ISSUE THREAD

**Issue by rickhg12hs** [Byron/dua-cli#89]
It would be useful to have the option of displaying the date and time of each directory/file, for example, while monitoring growing/shrinking files, etc.

**Comment by Byron** [comment]
I assume you mean file and directory modification dates are the ones to be shown, and agree that this could be a nice feature to have.

**Comment by gosuwachu** [comment]
FYI, this has been essentially implemented in #179

**Comment by Byron** [comment]
I agree, there is now one date/time that can be displayed, which matches the description of the issue.
